pages/article/[id].js

```diff
@@ -284,40 +317,41 @@ function ArticlePage() {
                 >{t`First reported ${timeAgoStr} ago`}</span>
               )}
             </Box>
-            <Divider classes={{ root: classes.divider }} />
-            <Box py={4} overflow="hidden">
-              {nl2br(
-                linkify(text, {
-                  props: {
-                    target: '_blank',
-                  },
-                })
-              )}
-              <Hyperlinks hyperlinks={hyperlinks} />
-            </Box>
-            <ArticleCategories
-              articleId={article.id}
-              articleCategories={article.articleCategories}
-            />
-            <Trendline />
-            <Divider />
-            <footer>
-              {article.replyRequests.map((replyRequest, idx) => (
-                <ReplyRequestReason
-                  key={replyRequest.id}
-                  articleId={article.id}
-                  replyRequest={replyRequest}
-                  isArticleCreator={idx === 0}
-                />
-              ))}
-              <CreateReplyRequestForm
-                requestedForReply={article.requestedForReply}
+            <Box px={{ xs: '12px', md: '19px' }}>
+              <Box py={3} overflow="hidden">
+                {nl2br(
+                  linkify(text, {
+                    props: {
+                      target: '_blank',
+                    },
+                  })
+                )}
+                <Hyperlinks hyperlinks={hyperlinks} />
+              </Box>
+              <ArticleCategories
                 articleId={article.id}
-                onNewReplyButtonClick={() => {
-                  setShowForm(true);
-                }}
+                articleCategories={article.articleCategories}
               />
-            </footer>
+              <Trendline />
+              <Divider />
+              <footer>
+                {article.replyRequests.map((replyRequest, idx) => (
+                  <ReplyRequestReason
+                    key={replyRequest.id}
+                    articleId={article.id}
+                    replyRequest={replyRequest}
+                    isArticleCreator={idx === 0}
+                  />
+                ))}
+                <CreateReplyRequestForm
+                  requestedForReply={article.requestedForReply}
+                  articleId={article.id}
+                  onNewReplyButtonClick={() => {
+                    setShowForm(true);
```

Comment:
I add the scrollIntoView feature in `useEffect` on Desktop editor so that when either float or regular button is clicked the editor scroll into view, and avoid deep ref pass-down issue.